src/schema/v2/partner.ts

```diff
@@ -562,8 +567,9 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
             page,
             size,
             sort: args.sort,
-            status: args.status,
+            status: args.status || undefined,
```

Comment:
Why can't this be `false`?

Reply:
if you mean something like `status: args.status || false` it throws an error
![image](https://user-images.githubusercontent.com/79979820/116103371-3054f700-a6b8-11eb-8be8-5925651709bd.png)

Reply:
That answers my question yeah (I meant why the need to remove the arg vs let it hang as false). Weird though that a boolean flag can be true but not false :(